.github/CODEOWNERS

```diff
@@ -130,85 +226,127 @@
 /core/domain/recommendations*.py @seanlip
 /core/domain/search_services*.py @kevinlee12
 /core/templates/dev/head/pages/library/ @kevinlee12
+/core/templates/dev/head/services/ConstructTranslationIdsService.ts @kevinlee12
+/core/templates/dev/head/services/SearchService*.ts @kevinlee12
 
 
 # Navigation bar project.
 /core/templates/dev/head/domain/sidebar/ @nithusha21
 /core/templates/dev/head/components/create_button/ @nithusha21
 /core/templates/dev/head/components/side_navigation_bar/ @nithusha21
 /core/templates/dev/head/components/top_navigation_bar/ @nithusha21
+/core/templates/dev/head/services/NavigationService*.ts @nithusha21
 
 
 # New structures team.
 # Question project.
+/core/controllers/practice_sessions*.py @vinitamurthi
 /core/controllers/question*.py @aks681
-/core/domain/question*.py  @aks681
+/core/domain/question*.py @aks681
 /core/storage/question/ @aks681
+/core/templates/dev/head/components/QuestionCreationService*.ts @aks681
 /core/templates/dev/head/domain/question/ @aks681
+/core/templates/dev/head/pages/practice_session/ @vinitamurthi
 /core/templates/dev/head/pages/question_editor/ @aks681
 /core/templates/dev/head/pages/questions_list/ @aks681
 
```

Comment:
Can we add question_player in this with me as the code owner? It was added in #6620

Reply:
Thanks for raising this @vinitamurthi, I think within last  28 days we have added many new files so I'll add all those files/folders before merging this PR! (otherwise, the lint test will start failing on develop.)

Currently, I've added a label `PR: don't merge - HAS MERGE CONFLICTS` to avoid merge.

Reply:
Done!